### PR TITLE
Added Basic Routes

### PR DESCRIPTION
**Pull request by athar05**
Added routes for Home, Rules, Quiz and Results pages using react router dom.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,13 +3,17 @@ import Home from "./pages/Home";
...
+import { Routes, Route } from "react-router-dom";
+
 function App() {
   return (
     <div className="App">
-      <Home />
-      <Rules />
-      <Result />
-      <Quiz />
+      <Routes>
```
Comment (raieegulhane) [inline src/App.js]: You can put routes into a separate file and import that component here for cleaner code

**Review by raieegulhane — COMMENTED** [review]
Please check the comment, else looks good.

File: src/App.js (hunk 1 above)
Comment (athar05) [reply src/App.js]: Thanks for the sugession. Will implement it.

**Review by sakshi006 — APPROVED** [review]
Good to merge

**Pull request merged**